test/lib/linter/rules/snapshots/BestPractices.ts.md

```diff
@@ -0,0 +1,271 @@
+        warningCount: 0,
+      },
+    ]
+
+## General: Negative_2/Component.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Negative_2/Component.js',
+        messages: [],
+        warningCount: 0,
+      },
+    ]
+
+## General: Negative_3/Component.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Negative_3/Component.js',
+        messages: [],
+        warningCount: 0,
+      },
+    ]
+
+## General: Negative_4/Component.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Negative_4/Component.js',
+        messages: [],
+        warningCount: 0,
+      },
+    ]
+
+## General: Negative_5/Component.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Negative_5/Component.js',
+        messages: [],
+        warningCount: 0,
+      },
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Negative_5/Component.js',
+        messages: [],
+        warningCount: 0,
+      },
+    ]
+
+## General: Negative_6/Component.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Negative_6/Component.js',
+        messages: [],
+        warningCount: 0,
+      },
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Negative_6/Component.js',
+        messages: [],
+        warningCount: 0,
+      },
+    ]
+
+## General: Positive_1/Component.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 1,
+        fatalErrorCount: 0,
+        filePath: 'Positive_1/Component.js',
+        messages: [
+          {
+            column: 9,
+            fatal: undefined,
+            line: 4,
+            message: 'Root View and Routing are not configured to load targets asynchronously',
+            messageDetails: 'Use Asynchronous Loading (https://ui5.sap.com/1.120/#/topic/676b636446c94eada183b1218a824717). Implement sap.ui.core.IAsyncContentCreation interface in Component.js or set async flags for "sap.ui5/routing/config" and "sap.ui5/rootView" in the manifest.json',
+            ruleId: 'ui5-linter-no-sync-loading',
+            severity: 2,
+          },
+        ],
+        warningCount: 0,
+      },
+    ]
+
+## General: Positive_2/Component.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Positive_2/Component.js',
+        messages: [
+          {
+            column: 9,
+            fatal: undefined,
+            line: 4,
+            message: 'Remove the async flag for "sap.ui5/rootView" from the manifest',
```

Comment:
What about a more positive message here, that also explains **why** the flag can/should be removed?
Without seeing the reference to `IAsyncContentCreation` I doubt that users would understand why they should remove an async flag, as all of our other documentation and best practices tell developers to make use of async flags.

Reply:
Done